torch/csrc/jit/passes/quantization/insert_quant_dequant.cpp

```diff
@@ -340,6 +342,8 @@ Node* insertEmbeddingBagOps(Node* observer, const std::string& op_name) {
     for (auto i = 1; i < inputs_size - 1; ++i) {
       qembedding_bag_inputs.push_back(embedding_bag_inputs[i]);
     }
+    // Set the sparse field to 0 for inference.
```

Comment:
optional: should we mention that this is for sparse gradients (as the pr summary reads), just to make it clear on why?